src/scripts/install-openshift-cli.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eox pipefail
+
+TEMPDIR=$(mktemp -d)
+
+# WARNING: these will only work on x86_64
+
+# https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html
+curl -L --insecure https://downloads-openshift-console.apps.ocp-virt.prod.psi.redhat.com/amd64/linux/oc.tar --output-dir "$TEMPDIR"
+# https://docs.openshift.com/container-platform/4.13/virt/virt-using-the-cli-tools.html
+curl -L --insecure https://hyperconverged-cluster-cli-download-openshift-cnv.apps.ocp-virt.prod.psi.redhat.com/amd64/linux/virtctl.tar.gz --output-dir "$TEMPDIR"
```

Comment:
None of these are accessible outside of VPN so I think this ain't going to work. 

However I don't know where else we can get these binaries from.  There seems to be no RPMs available on Fedora/CentOS and publicly accessible download pages for `oc.tar.gz` generate a temporary time-stamped URL which expires in about 2 hours.